tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/bluetooth.py

```diff
@@ -35,15 +35,21 @@ def device_name(self):
     def enable_bluetooth(self):
         self.marionette.find_element(*self._bluetooth_label_locator).tap()
         checkbox = self.marionette.find_element(*self._bluetooth_checkbox_locator)
-        Wait(self.marionette).until(expected.element_selected(checkbox))
+        Wait(self.marionette).until(lambda m: self.is_custom_element_checked(checkbox))
         rename_device = self.marionette.find_element(*self._rename_my_device_button_locator)
         Wait(self.marionette).until(expected.element_enabled(rename_device))
 
+    def disable_bluetooth(self):
```

Comment:
Actually... I plan to use it for my RTL testing.  I need a way to turn off normally, so I can take the screenshot when it's off.  Should I add it on that PR?  (Which I am working on at the moment as well)